package.json

```diff
@@ -14,29 +14,33 @@
     "build": "tsc",
     "test": "vitest --exclude dbos-demo-apps"
   },
+  "keywords": [
+    "eslint",
+    "eslintplugin"
+  ],
   "dependencies": {
-    "@types/node": "^20.14.6",
+    "@types/node": "^20.14.12",
     "eslint-plugin-no-secrets": "^1.0.2",
-    "eslint-plugin-security": "^3.0.1",
-    "ts-morph": "22.0",
-    "typescript": "5.4.2",
-    "typescript-eslint": "7.14",
-    "@typescript-eslint/utils": "7.14",
-    "@typescript-eslint/parser": "7.14",
-    "@typescript-eslint/eslint-plugin": "7.14"
+    "eslint-plugin-security": "^3.0.1"
   },
   "peerDependencies": {
-    "typescript": "5.4.2"
+    "@typescript-eslint/eslint-plugin": "^7.17.0",
+    "@typescript-eslint/parser": "^7.17.0",
+    "@typescript-eslint/utils": "^7.17.0",
+    "eslint": "^8.57.0",
+    "ts-morph": "^23.0.0",
+    "typescript": "^5.5.4",
```

Comment:
Do we need these two in devDependencies?

Reply:
Added them